packages/dashboard/src/components/Card/examples/Card.Chart.MultipleLineChart.Example.tsx

```diff
@@ -103,6 +103,11 @@ export class MultipleLineChartExample extends React.Component<{}, {}> {
       headerText: 'Header Text '
     };
 
-    return <Card cardFrameContent={cardFrameContent} header={header} cardContentList={contentAreaList} cardSize={CardSize.mediumWide} />;
+    return (
+      // tslint:disable-next-line
```

Comment:
Nit: only disable the jsx-ban-props rule here rather than disabling all tslint rules: `// tslint:disable-next-line:jsx-ban-props`

Reply:
changed tslint comments